src/display/components/total_bandwidth.rs

```diff
@@ -21,9 +22,12 @@ impl<'a> TotalBandwidth<'a> {
                 Color::Green
             };
 
+            let interface = self.interface_name.as_deref().unwrap_or("[N/A]");
+            
             [Text::styled(
                 format!(
-                    " Total Rate Up / Down: {} / {} {}",
+                    " Interfaces {} / Total Rate Up / Down: {} / {} {}",
+                    DisplayString(interface),
```

Comment:
I'm not sure I understand... why do we need `DisplayString` here? Why not input the interface name directly?

Also, could we make the display:
* if there is a specific interface passed in the options, we will display `<interface_name>: Total Rate Up / Down: {} / {} {}`
* if there is no interface (meaning we're listening on all interfaces), we'll just display the previous string without the interfaces

What do you say?

Reply:
@imsnif  Sure. I've made all changes.
Travic CI build failing every time. Do you know the reason?
Also, I don't know why, but on my machine all tests are failing every time, even on master branch.

Reply:
As for travis: you can see the output of the snapshot tests (these are essentially tests that run the app under certain situations take a "snapshot" of the terminal screen and see that the output is what they expect - these should be quite readable, so you can look at the travis output linked in this PR and check them out).

As for your local tests: if you tell me more about why they are failing, maybe we can try and debug it?

Reply:
@imsnif After `cargo test` command,  `insta` package creates new files.
![tests1](https://user-images.githubusercontent.com/16646733/72686141-bcf62a00-3af1-11ea-8f7c-c5fd8e8d59a0.png)
![tests2](https://user-images.githubusercontent.com/16646733/72686142-bd8ec080-3af1-11ea-86e6-9159120ba718.png)

Complete [log](https://gist.github.com/SerhiiBilyk/1c09a487e15ee9146df6c710d99462af)
Part of the log with color for readability
![tests3](https://user-images.githubusercontent.com/16646733/72686323-aea90d80-3af3-11ea-8218-5525430176e2.png)

P.S. What do you think about my last changes ? Is it ok?
P.S.S. Sorry for taking you time, I'm really want to help, but I have not a lot experience in Rust, only learning.

Reply:
No worries friend! We all have to start sometime, right? Thanks for taking the time to work on this. :) The snapshots should be part of the repository, so it's odd that the tests are creating them instead. I'd try to look in that direction.
I'll take a look at your changes here and in the other PR in the next few days.